lib/access_import/accessors/accessor.rb

```diff
@@ -2,12 +2,11 @@
 
 class Accessor
   include AccUtils
-  attr_reader :records
+  attr_accessor :all
```

Comment:
The reader part will be overwritten by `def all` and the writer part is not used anywhere.